eslint/react/index.js

```diff
@@ -17,12 +17,4 @@ module.exports = {
       version: 'detect',
     },
   },
-  overrides: [
-    {
-      files: ['**/tests/**/*.{js,jsx,ts,tsx}'],
-      rules: {
-        'react/jsx-props-no-spreading': 'off',
```

Comment:
This is not enabled by default, so it's silly to override the config to turn it off.

It also seems like ESLint's `calculateConfigForFile` might be ignoring `overrides` when used with a `baseConfig` option, which doesn't make a lot of sense. They seem to be pretty aggressively ignoring issues that have anything to do with ESLint configuration, as Version 8 of ESLint is going to totally re-vamp how configuration works.